packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown.tsx

```diff
@@ -79,7 +121,10 @@ export class FbFieldDropdown {
     if (this.dataProvider && this.dataProvider.length > 0) {
       this.errorType = '';
       const strDuplicateErrorKey = i18nText('errors.duplicate');
-      const arrChoices = deepCloneObject(this.dataProvider);
+      const clonedChoices = deepCloneObject(this.dataProvider);
+      const arrChoices = this.level
```

Comment:
can we add (this.isDependentField && this.level) here

Reply:
Done.